fw/http.c

```diff
@@ -1694,9 +1704,9 @@ tfw_http_req_zap_error(struct list_head *eq)
 
 	list_for_each_entry_safe(req, tmp, eq, fwd_list) {
 		list_del_init(&req->fwd_list);
-		if ((TFW_MSG_H2(req) && req->stream)
-		    || (!TFW_MSG_H2(req)
-			&& !test_bit(TFW_HTTP_B_REQ_DROP, req->flags)))
+		if ((TFW_MSG_H2(req) && req->stream) ||
```

Comment:
Unnecessary changes, please revert it and LGTM. I also implement test which reproduce this bug on master branch https://github.com/tempesta-tech/tempesta-test/pull/610